app/_src/gateway/kong-enterprise/datakit/get-started.md

```diff
@@ -30,7 +30,7 @@ Add the following configuration to a `kong.yaml` file:
 ```yaml
 _format_version: "3.0"
```

Comment:
GH wont let me comment but this is what my services block looked like after indentation
```
services:
- url: http://httpbin.org
  name: my-service
  routes:
  - name: my-route
    paths:
    - /
    strip_path: true
```

Reply:
🤔 this is exactly the same as the indentation on the example already.